scripts/upload-rubbernecker-secrets.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -eu
+
+export PASSWORD_STORE_DIR=${RUBBERNECKER_PASSWORD_STORE_DIR}
+
+PIVOTAL_API_KEY=$(pass "pivotal/tracker_token")
+PAGERDUTY_API_TOKEN=$(pass "pagerduty/rubbernecker_api_token")
+
+SECRETS=$(mktemp secrets.yml.XXXXXX)
+trap 'rm "${SECRETS}"' EXIT
```

Comment:
I'm not sure that string interpolation will happen in this situation—it doesn't happen with either ZSH or Bash so I suspect whatever `/bin/sh` is in this case won't do either.

Reply:
`trap` is special. It expands the variable at the time the trap executes instead of when it's defined.

I've tested this locally, and the file is cleaned up correctly. This also follows the same pattern that the other secret upload scripts use.

Reply:
Hmmm, interesting. Sorry about this.